pkg/csi_driver/controller.go

```diff
@@ -950,6 +950,10 @@ func (s *controllerServer) CreateSnapshot(ctx context.Context, req *csi.CreateSn
 		if err != nil {
 			return nil, err
 		}
+		// Append extra labels from the command line option
+		for k, v := range s.config.extraVolumeLabels {
```

Comment:
> Can you add a unit test that validates propagation of labels? https://github.com/kubernetes-sigs/gcp-filestore-csi-driver/blob/master/pkg/csi_driver/controller_test.go#L1529

I hadn't added any because labels are just merged. I will add a test case for labels.

Yeah, [labels processed](https://github.com/kubernetes-sigs/gcp-filestore-csi-driver/blob/master/pkg/csi_driver/controller.go#L295-L303) for adding to instance resource also does the same. Should this be kept consistent with PD driver?

Reply:
@pwschuurman Suggestions incorporated. PTAL.

Reply:
Thanks, LGTM